{
    "PackageVulnerabilityID": "1",
    "DisclosureDate": "2021-01-07",
    "AffectedVersionRange": "*",
    "Summary": "Arbitrary Code Execution",
    "Contributor": {
        "Discloser": "61599526",
        "Fixer": "43377443"
    },
    "Package": {
        "Registry": "pip",
        "Name": "aiocache",
        "URL": "https://pypi.org/project/aiocache",
        "Downloads": "5615"
    },
    "CWEs": [
        {
            "ID": "94",
            "Description": "Arbitrary Code Execution"
        }
    ],
    "CVSS": {
        "Version": "3.1",
        "AV": "",
        "AC": "",
        "PR": "",
        "UI": "",
        "S": "",
        "C": "",
        "I": "",
        "A": "",
        "E": "",
        "RL": "",
        "RC": "",
        "Score": "8.3"
    },
    "CVEs": [
        ""
    ],
    "Repository": {
        "URL": "https://github.com/aio-libs/aiocache",
        "Codebase": [
            "Python"
        ],
        "Owner": "aio-libs",
        "Name": "aiocache",
        "Forks": "73",
        "Stars": "476"
    },
    "Permalinks": [
        "https://drive.google.com/file/d/1Zf-OEiLoDx4wEDabElhWaW1AQgVG5bAO/view?usp=sharing"
    ],
    "References": [
        {
            "Description": "",
            "URL": ""
        }
    ],
    "PrNumber": "1493",
    "FixSubmissionCount": 1
}